src/Ztm.Zcoin.NBitcoin/ZcoinTxIn.cs

```diff
@@ -31,26 +36,28 @@ public bool IsSigmaSpend
             {
                 return prevout.Hash == uint256.Zero
                     && prevout.N >= 1
-                    && scriptSig.Length > 0
-                    && scriptSig.ToBytes()[0] == OpSigmaSpend;
+                    && IsType(ZcoinOpCode.OpSigmaSpend);
             }
         }
 
         public bool IsZerocoinRemint
         {
             get
             {
-                return prevout.IsNull && scriptSig.Length > 0 && scriptSig.ToBytes()[0] == OpZerocoinToSigmaRemint;
+                return prevout.IsNull && IsType(ZcoinOpCode.OpZerocoinToSigmaRemint);
             }
         }
 
         public override ConsensusFactory GetConsensusFactory()
         {
             return this.consensusFactory;
         }
+    }
 
-        public static readonly byte OpZerocoinSpend = 0xc2;
-        public static readonly byte OpSigmaSpend = 0xc4;
-        public static readonly byte OpZerocoinToSigmaRemint = 0xc8;
+    enum ZcoinOpCode : byte
+    {
+        OpZerocoinSpend = 0xc2,
```

Comment:
We don't need to prefix it with `Op` anymore due to it is already a member of `enum`.